pyanaconda/modules/common/errors/general.py

```diff
@@ -38,3 +38,8 @@ class InvalidValueError(AnacondaError):
 class UnsupportedValueError(AnacondaError):
     """Value passed is not supported."""
     pass
+
+
+class KickstartProcessingWarning(UserWarning):
```

Comment:
Btw. `pyanaconda.core.kickstart` would be a good place for this exception. `pyanaconda.modules.common.errors` contains only exception that are mapped on DBus errors.

Reply:
Thanks!